src/components/Firealerts/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { Suspense } from "react";
+import { useFirealertsTriggers } from "./api/internal/useFirealertsTriggers";
+import { EmulatorDisabled } from "../common/EmulatorDisabled";
+import { Spinner } from "../common/Spinner";
+import { useIsEmulatorDisabled } from "../common/EmulatorConfigProvider";
+import { FirealertsProvider } from "./api/useFirealerts";
+import { Switch } from "react-router";
+import { Route } from "react-router-dom";
+import { FirealertsForm } from "./Form/FirealertsForm";
+
+export const FirealertsRoute: React.FC = () => {
+  return (<Suspense fallback={<FirealertsRouteSuspended />}>
+    <HydrateFirealerts>
+      <Switch>
+        <Route path="/firealerts">
+          <FirealertsForm />
+        </Route>
+      </Switch>
+    </HydrateFirealerts>
+  </Suspense>
+  );
+};
+
+
+const HydrateFirealerts: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => {
+  const functions = useFirealertsTriggers();
+  return (
+    <FirealertsProvider triggers={functions}>{children}</FirealertsProvider>
+  )
+};
+const FirealertsRouteSuspended: React.FC<
+  React.PropsWithChildren<unknown>
+> = () => {
+  const isDisabled = useIsEmulatorDisabled('eventarc');
+  return isDisabled ? (
+    <EmulatorDisabled productName="FireAlerts" />
```

Comment:
Dumb question, but is FireAlerts considered one word or two words for naming purposes? e.g. component and file names

Reply:
I believe (and have been operating under the assumption) that it is one word. I think it is a little confusing for component naming conventions w/ camel case because the one word is capitalized as "FireAlerts".

I am open to any thoughts/suggestions on how files & components should be named.

Reply:
Thanks for the clarification. Totally fine either way as long as it is consistent.